overwatch-rs/src/services/handle.rs

```diff
@@ -21,7 +21,7 @@ pub struct ServiceHandle<S: ServiceData> {
     /// Handle to overwatch
     overwatch_handle: OverwatchHandle,
     settings: SettingsUpdater<S::Settings>,
-    initial_state: S::State,
+    pub initial_state: S::State,
```

Comment:
Is there a specific reason this is public?

Reply:
Slipped in, thanks for the catch.